model/project_parser.go

```diff
@@ -85,6 +85,7 @@ type ParserProject struct {
 	TaskAnnotationSettings *evergreen.AnnotationsSettings `yaml:"task_annotation_settings,omitempty" bson:"task_annotation_settings,omitempty"`
 	BuildBaronSettings     *evergreen.BuildBaronSettings  `yaml:"build_baron_settings,omitempty" bson:"build_baron_settings,omitempty"`
 	PerfEnabled            *bool                          `yaml:"perf_enabled,omitempty" bson:"perf_enabled,omitempty"`
+	Aliases                []ProjectAlias                 `yaml:"aliases,omitempty" bson:"aliases,omitempty"`
```

Comment:
We discussed in slack potentially breaking this up into different kinds of aliases and merging this ourselves (GithubAliases, GitTagAliases, etc) -- did we decide not to do this? In which case we'll just need to be very clear in our documentation what aliases should be named to apply to these different things.